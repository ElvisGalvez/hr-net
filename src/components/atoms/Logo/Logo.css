.logo-container {
    display: flex;
    justify-content: center;
    align-items: center;
  }
  
  .logo {
    width: 80%;
    margin: 5px 0;
  }
  