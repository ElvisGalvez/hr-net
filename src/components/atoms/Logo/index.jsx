import React from 'react';
import './Logo.css';
import logoPath from '../../../assets/logos/wealth health logo.jpg';

const Logo = ({ isExpanded }) => (
  <div className="logo-container">
    <img 
      src={logoPath} 
      alt="Wealth Health Logo" 
      className={`logo ${isExpanded ? 'expanded' : 'collapsed'}`}
    />
  </div>
);

export default Logo;