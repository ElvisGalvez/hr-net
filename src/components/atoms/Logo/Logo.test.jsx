import React from 'react';
import renderer from 'react-test-renderer';
import Logo from './index'; 

describe('Logo', () => {
  it('renders correctly when expanded', () => {
    const tree = renderer
      .create(<Logo isExpanded={true} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly when collapsed', () => {
    const tree = renderer
      .create(<Logo isExpanded={false} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
