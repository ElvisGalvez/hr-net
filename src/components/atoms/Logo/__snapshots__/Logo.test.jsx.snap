// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly when collapsed 1`] = `
<div
  className="logo-container"
>
  <img
    alt="Wealth Health Logo"
    className="logo collapsed"
    src="wealth health logo.jpg"
  />
</div>
`;

exports[`Logo renders correctly when expanded 1`] = `
<div
  className="logo-container"
>
  <img
    alt="Wealth Health Logo"
    className="logo expanded"
    src="wealth health logo.jpg"
  />
</div>
`;
